load-testing/tests/relays_stress_test.go

```diff
@@ -217,8 +225,8 @@ type relaysSuite struct {
 
 // accountInfo contains the account info needed to build and send transactions.
 type accountInfo struct {
-	// keyName is the key name of the account available in the keyring used by the test.
-	keyName       string
+	// The address of the account available in the keyring used by the test.
+	address       string
 	accAddress    sdk.AccAddress
```

Comment:
We have `address` and `accAddress`. Do we need both?

Reply:
The reason why `accAddress` is kept is that `bank.NewMsgSend` and  `KeyRing.GetKeyByAddress` require `AccAddress` structs.

We can create them on the fly though. I'll give their removal a try since we can derive one from the other.